Blog: Writing Day project announcements

rosewms opened the pull request:
Hopefully we can get Jupyter and the other potential OSS project in there. 

I'd happily get this out on Friday, Sept 13, if possible. However, it would be great to get at least one of the other projects added - just to bulk out the day.

In an ideal world, this would go live no later than Monday, Sept 16.

<!-- readthedocs-preview writethedocs-www start -->
----
📚 Documentation preview 📚: https://writethedocs-www--2216.org.readthedocs.build/

<!-- readthedocs-preview writethedocs-www end -->

rosewms committed:
Blog: Writing Day project announcements

Draft

mxsasha commented on docs/conf/atlantic/2024/news/announcing-wd-projects.rst:
```diff
@@ -0,0 +1,35 @@
+
+Didn't get the chance to submit your project early? Fear not, you can submit it at any time between now and September 20 or you can bring it and submit it live during Writing Day. Day of project submissions are a time honored WTD tradition that we look forward to seeing continue.
+
+Get excited, get delighted, and get ready to meet our Writing Day projects for Write the Docs Atlantic 2024.
```
```suggestion
Get excited, get delighted, and get ready to meet our Writing Day projects for Write the Docs Atlantic 2024!

Writing Day is included in our conference tickets. Sales close Thursday, September 19, at 23:59 UTC, so `get your tickets soon <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_.
```

mxsasha reviewed (commented):
I think this should also have a ticket reminder, so that it's clear they need to get one, in time, added suggestion inline. Looks good otherwise! I'd prefer for this to go out on Friday, so that we don't bunch up the mails to the mailing list too much with #2205 pending.

rosewms committed:
Update docs/conf/atlantic/2024/news/announcing-wd-projects.rst

Co-authored-by: Sasha Romijn <github@mxsasha.eu>

rosewms committed:
Update announcing-wd-projects.rst

eirinikos committed:
Refine the WD blog post

Fix a few typos and move the ticket sale reminder (so that the  "get excited" bits can flow freely ;)

eirinikos committed:
Merge branch 'main' into wd-proj-announcements

rosewms committed:
Merge branch 'main' into wd-proj-announcements

mxsasha committed:
Update post date
